### ISSUE TITLE
Can't report an infection through the app

### ISSUE THREAD

**Issue by bedo2991** [corona-warn-app/cwa-app-android#5094]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name: Pixel 4a
- Android version: 12
- App version: 2.20.4

## Describe the bug
I got a QR code of a positive quick test result, which gets imported in the App via a link which looks like https://s.coronawarn.app?v=1#...
After I click on "Accept", I land on the page which asks me to "Request test certificate" or choose "No thank you", any of them I choose, I get a popup with "QR code is invalid", "The QR code is invalid or has been registered on your smartphone already. You will receive your test result from the testing center or laboratory regardless of the validity of the QR code. If you are diagnosed with coronavirus the public health authority will be notified through the legally prescrived channel and will contact you. If you have deleted the test in the app, you can restore it from the recycle bin".
The recycle bin does not contain this test.

## Steps to reproduce the issue
It can be, that when I first imported the test the connection switched from wifi to 4G, causing an interruption.

## Expected behaviour
I should be able to import the positive result and warn others.

## Additional context
I called the hotline to get a TAN, and they told me they only do it when I'll have a PCR test (in 1-2 days, very useful).
A friend of mine who uses iPhone got a positive quick test result from the same test center some days ago, and the warning worked fine for him.

**Comment by Ein-Tim** [comment]
There's probably not much that can be done to help you, maybe you can call the testing station, explain the situation and ask them to create a new QR code/link?

I hope you have a mild course of COVID & get well soon!

**Comment by mtwalli** [comment]
@bedo2991 It is only possible to register the test Only once in CWA either (Android or iOS versions) , subsequent trials to register the test will fail. Once it is marked on the server as registered It is not possible from the client side (App) to surpass this
